cv_next/components/ui/popupButtom.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+
+import { ReactNode, useState } from "react";
+
+interface PopupProps {
+  children: ReactNode;
+  clickable?: ReactNode;
+  disableButton?: boolean;
+  onClose?: () => void;
+}
+
+export default function PopupWrapper({
+  children,
+  clickable,
+  disableButton,
```

Comment:
I don't really get this `disableButton` prop. Why not simply use the `disabled` attribute ok a `<button>` element?

Reply:
its for disabling the button, i don't understand

Reply:
Do you mean that instead of a div with onClick that is "defended" using the attribute, use the attribute for disabled?

Reply:
I'm not sure I understand this component. Are you wrapping a button (that you're expecting that get sent as prop) and override its functionality with the open/close mechanism you're doing here?
A. Again, why not use the `disabled` attribute of a button? (yes Arad, you understood correctly)
B. If you want to create a generic popup, why not simply add this button functionality as part of the popup code instead of passing it through prop?

Reply:
If I understand correctly, this is a combination of the popup and the button that opens it
About A it'll be fixed, about B I think that the point is to allow anything we want to serve as the "popup trigger"

Reply:
Uploaded the fix mentioned in A